crates/store/re_log_types/src/index/timeline.rs

```diff
@@ -65,14 +64,30 @@ impl Timeline {
         }
     }
 
+    /// For relative times (e.g. seconds since start).
     #[inline]
-    pub fn new(name: impl Into<TimelineName>, typ: TimeType) -> Self {
+    pub fn new_duration(name: impl Into<TimelineName>) -> Self {
         Self {
             name: name.into(),
-            typ,
+            typ: TimeType::Time,
+        }
+    }
+
+    /// For absolute timestamps.
+    #[inline]
+    pub fn new_timestamp(name: impl Into<TimelineName>) -> Self {
+        Self {
+            name: name.into(),
+            typ: TimeType::Time,
         }
     }
 
+    #[deprecated(note = "Use `Timeline::new_duration` or `new_timestamp` instead")]
```

Comment:
here and elsewhere, should fill ou the `since` field so it's easier for us to keep track of when to remove deprecated methods